internal_test.go

```diff
@@ -1,10 +1,35 @@
 package circonusllhist
```

Comment:
why isnt this file circonusllhist_test.go?

Reply:
renamed in latest commit to circonusllhist_test.go